src/components/RepoBranchesDrawer/RepoBranchesDrawer.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+
+import GitHubStore from '@store/GitHubStore';
+import { BranchItem, RepoItem } from '@store/GitHubStore/types';
+import { Drawer } from 'antd';
+
+export type RepoBranchesDrawerProps = {
+  selectedRepo: RepoItem | null;
+  onClose: (e: any) => void;
+};
+
+const RepoBranchesDrawer: React.FC<RepoBranchesDrawerProps> = ({
+  selectedRepo,
+  onClose,
+}) => {
+  const [branchesList, setBranchesList] = useState<BranchItem[]>([]);
+
+  React.useEffect(() => {
+    if (!selectedRepo) return;
+
+    const gitHubStore = new GitHubStore();
+    gitHubStore
+      .getBranchesList({
+        owner: selectedRepo.owner.login,
+        repo: selectedRepo.name,
+      })
+      .then((result) => {
+        if (result.success) {
+          setBranchesList(result.data);
+        } else setBranchesList([]);
+      });
+  }, [selectedRepo]);
+
+  return (
+    <Drawer
+      onClose={onClose}
+      visible={!!selectedRepo}
+      title={<div>Ветки репозитория {selectedRepo?.name}</div>}
```

Comment:
а зачем оборачивать в `div`? не получиться ли просто с 
```suggestion
      title={`Ветки репозитория ${selectedRepo?.name}`}
```